code/game/objects/structures/bookcase.dm

```diff
@@ -7,6 +7,25 @@
 	density = TRUE
 	opacity = TRUE
 
+/obj/structure/bookcase/deconstruct(disassembled)
+	var/metal = /obj/item/stack/sheet/metal
+	new metal(src.loc)
+	. = ..()
+
+/obj/structure/bookcase/attack_alien(mob/living/carbon/xenomorph/xeno)
+	if(xeno.a_intent == INTENT_HARM)
+		if(unslashable)
+			return
+		xeno.animation_attack_on(src)
+		playsound(src.loc, 'sound/effects/metalhit.ogg', 25, 1)
```

Comment:
```suggestion
		playsound(loc, 'sound/effects/metalhit.ogg', 25, 1)
```

Reply:
You undid this change with https://github.com/cmss13-devs/cmss13/pull/6213/commits/6fcc9b3e4033b2b90f14796e6a0ae35b6fcbfd44